src/ZZMatrix-linalg.jl

```diff
@@ -282,7 +282,22 @@ end
 
 #adaptive rational_reconstruction: if solution is unbalanced with
 #denominator smaller than numerator
-function induce_rational_reconstruction(a::ZZMatrix, b::ZZRingElem; ErrorTolerant ::Bool = false)
+@doc raw"""
+    induce_rational_reconstruction_split(a::ZZMatrix, b::ZZRingElem; ErrorTolerant ::Bool = false, Unbalanced::Bool = true) -> Bool, ZZMatrix, ZZRingElem
```

Comment:
```suggestion
    induce_rational_reconstruction_split(a::ZZMatrix, b::ZZRingElem; error_tolerant ::Bool = false, unbalanced::Bool = true) -> Bool, ZZMatrix, ZZRingElem
```
for consistency with the rest of the codebase, the kwargs should be snake_case as well (here and in all other places these are introduced)